src/main/java/org/prebid/server/spring/config/bidder/BidderConfiguration.java

```diff
@@ -11,17 +11,29 @@
 import org.prebid.server.bidder.Usersyncer;
 import org.prebid.server.vertx.http.HttpClient;
 
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
 public abstract class BidderConfiguration {
 
     private static final String ERROR_MESSAGE_TEMPLATE_FOR_DISABLED = "%s is not configured properly on this "
             + "Prebid Server deploy. If you believe this should work, contact the company hosting the service "
             + "and tell them to check their configuration.";
 
+    private static final String ERROR_MESSAGE_TEMPLATE_FOR_DEPRECATED = "%s has been deprecated and is no "
+            + "longer available. Use %s instead.";
+
     protected BidderDeps bidderDeps(HttpClient httpClient, HttpAdapterConnector httpAdapterConnector) {
         final String bidderName = bidderName();
         final MetaInfo metaInfo = createMetaInfo();
         final boolean enabled = metaInfo.info().isEnabled();
 
+        final Map<String, String> deprecatedNameToError = deprecatedNames().stream()
```

Comment:
Is there a good reason to construct error message here? Isn't it natural to pass `deprecatedNames` to BidderDeps and let it create message instead?

Reply:
We did it as well as for disabled bidders - construct error messages in configuration, so just to be in same style.

Reply:
For disabled bidder there is a good reason to place it here - message is shared between disabled bidder and adapter, otherwise it belonged to disabled bidder.